EnglishFilter/sections/specific.txt

```diff
@@ -158,6 +158,7 @@ hdporncomics.com###sidebar-right > #text-3
 hdporncomics.com###sidebar-right > #custom_html-9
 ||analyticsindiamag.com/wp-content/uploads/2019/02/Blue_03.jpg
 analyticsindiamag.com##.textwidget > p > a[rel="nofollow external noopener noreferrer"] > img
+||activistpost.com/OeabcdRPf/VqFr/bcaeqn.jpg
```

Comment:
I'm afraid that the path/name of this script may be changed sometimes, so maybe something like this?
`||activistpost.com/*/*.jpg$script`

Reply:
agree with it